src/main/java/org/littil/api/guestTeacher/api/GuestTeacherModuleResource.java

```diff
@@ -97,8 +95,8 @@ public Response deleteGuestTeacherModule(@Parameter(name = "guestTeacher_id", re
     @APIResponse(
             responseCode = "401",
             description = "Current user is not owner of this guestTeacher"
-    )public Response saveGuestTeacherModule(@Parameter(name = "guestTeacher_id", required = true) @PathParam("guestTeacher_id") UUID guestTeacherId,  @NotNull @Valid Module module) {
```

Comment:
The `Parameter` annotation was left-over here, only `PathParam` is required in this scenario.